src/Swift.Bindings/tests/IntegrationTests/FunctionalTests/Structs/StructsTests.swift

```diff
@@ -257,3 +257,18 @@ public func sumArray(array: Array<Int32>) -> Int32
 {
     return array.reduce(0, +)
 }
+
+public func getString() -> String
```

Comment:
I understand this is here as it requires some swift code. But I dont think the label 'struct tests' describes it very well. Maybe we should create a new test file?